src/pyspiffe/bundle/jwt_bundle/jwt_bundle.py

```diff
@@ -60,10 +61,10 @@ def get_jwt_authority(self, key_id: str) -> Optional[_PUBLIC_KEY_TYPES]:
             None if the key_id is not found.
 
         Raises:
-            ValueError: When key_id is not valid (empty or None).
+            ValidationError: When key_id is not valid (empty or None).
```

Comment:
For this kind of error, where an invalid argument was passed, I would throw an error named `IllegalArgumentError`, so it is immediately clear to the caller that some arguments were missing or empty.

Reply:
IllegalArgument seems java like.... ValueError seems more common in python codes for invalid arguments?

Reply:
Maybe `InvalidArgumentError`, `ArgumentError`, or `InvalidValueError`.  Something that doesn't have the word `Validation` in it.

Reply:
My vote is for `ArgumentError`.